docs/geolocation.md

```diff
@@ -0,0 +1,234 @@
+
+- Run migrations to add the new field `coordnites`:
+
+```bash
+python manage.py makemigrations
+python manage.py migrate
+```
+
+#### Step 3: Data Migration to Populate New Field
+
+- Create a **data migration** to copy data from the old `point` field to the new `coordinates` field.
+
+Example migration snippet:
+
+```python
+import json
+import logging
+
+from django.contrib.gis.geos import GEOSGeometry
+from django.db import migrations
+from django.contrib.gis.geos import Point
+
+logger = logging.getLogger(__name__)
+
+
+def migrate_point_to_coordinates(apps, schema_editor):
+    MapIdea = apps.get_model('a4_candy_mapideas', 'MapIdea')
+    for idea in MapIdea.objects.exclude(point__isnull=True):
+        try:
+	    # Handle case where point is already parsed (dict) or still a string
+	    point_data = idea.point
+	    if isinstance(point_data, str):
+                point_data = json.loads(point_data)
+                point_data = dict(point_data)
+
+	    # Extract geometry and properties
+	    geometry = point_data.get("geometry", {})
+	    properties = point_data.get("properties", {})
+	    if not geometry:
+		logger.warning(
+		    "error migrating point of idea " + idea.name + ": " + str(point_data)
+		)
+		continue
+
+	    # Create GEOSGeometry from coordinates
+	    if geometry.get("type") == "Point" and "coordinates" in geometry:
+		geojson = {"type": "Point", "coordinates": geometry["coordinates"]}
+		point = GEOSGeometry(json.dumps(geojson), srid=4326)
+		# Update all fields
+		MapIdea.objects.filter(id=idea.id).update(
+		    coordinates=point,
+		    street_name=properties.get("strname", ""),
+		    house_number=properties.get("hsnr", ""),
+		    zip_code=properties.get("plz", ""),
+		)
+
+	except (ValueError, TypeError, KeyError, json.JSONDecodeError) as e:
+	    logger.warning(f"Skipping {project.id} {project.name}: {str(e)}")
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('a4_candy_mapideas', 'previous_migration'),
+    ]
+
+    operations = [
+        migrations.RunPython(migrate_point_to_coordinates),
+    ]
+```
+
+- Run the migration:
+
+```bash
+python manage.py migrate
+```
+
+#### Step 4: Remove the Old `map_fields.PointField`
+
+- After verifying the new field is populated and working correctly, remove the old `point` field from the model.
+
+- Create and apply a migration to drop the old field with `python manage.py makemigrations` and `python manage.py migrate`
+
+#### Step 5: Add new PointField
+
+- If all data is migrated, add a new Geo GIS field named point and copy data from coordinates. We need to do this because a simple renaming a GeoDjango PointField (or any GIS field) directly via migrations often fails due to spatial database complexities and Django's migration detection limitations. E.g: GIS fields like PointField require special database metadata (e.g., PostGIS geometry_columns table). A simple column rename won’t update these registries, leading to broken spatial queries.
+
```

Comment:
@partizipation @vellip FYI, I doubled check and we cannot just rename a GEO GIS field for the reasons I mention here above.